Makefile

```diff
@@ -1,125 +1,42 @@
 
 # Provide tag as:
 # GIT_COMMIT_HASH=$(git rev-parse --short HEAD)
-ENV_DATABASE_HOST ?= "172.17.0.1"
-ENV_DATABASE_PORT ?= 5432
-ENV_SERVER_PORT ?= 80
-# Provide password as:
-# ENV_DATABASE_PASSWORD=password
-
 NODE_PORT ?= 3001
 SERVER_ORIGIN ?= "http://localhost:3001"
-API_BASE_URL ?= "http://galactic-sovereign-service:80/v1"
 USER_API_BASE_URL ?= "http://user-service:80/v1/users"
-
-# https://docs.docker.com/network/drivers/bridge/
-DOCKER_NETWORK_BRIDGE_NAME ?= "corp-network"
-
-RESTART_RETRIES_COUNT ?= 5
+GALACTIC_SOVEREIGN_API_BASE_URL ?= "http://galactic-sovereign-service:80/v1/galactic-sovereign"
```

Comment:
Cleaned a bit the logic in this file: some of it was not up to date and it seems like can't really start a docker image without the cluster around it. So this file will only keep commands to build the docker containers for each service.